ocictl/cmd/build.go

```diff
@@ -35,12 +35,13 @@ It can run a build in both docker and buildah varieties.
 `
 
 type buildCmd struct {
```

Comment:
can you run `go fmt`?

Reply:
We should run it as part of any local commit hook (husky?)

Reply:
created issue #56